Website/HTML/pulse_poem.html

```diff
@@ -1,236 +1,183 @@
-
-                  <div id="lineNum31" class="line">my bones.</div>
-               </div>
-
-
-               <div id="stanzaNum7" class="stanza">
-
-                  <div id="lineNum32" class="line heartbeat_line">Heartbeat.</div>
-
-                  <div id="lineNum33" class="line">Hammering down</div>
-
-                  <div id="lineNum34" class="line">to the molten</div>
-
-                  <div id="lineNum35" class="line">core.</div>
-
-                  <div id="lineNum36" class="line heartbeat_line">Heartbeat.</div>
-
-                  <div id="lineNum37" class="line">Fracturing of the</div>
-
-                  <div id="lineNum38" class="line">crust.</div>
-
-                  <div id="lineNum39" class="line heartbeat_line">Heartbeat.</div>
-
-                  <div id="lineNum40" class="line">Rhythmic</div>
-
-                  <div id="lineNum41" class="line">annihilation.</div>
-               </div>
-
-
-               <div id="stanzaNum8" class="stanza">
-
-                  <div id="lineNum42" class="line heartbeat_line">Heartbeat</div>
-
-                  <div id="lineNum43" class="line heartbeat_line">beat</div>
-
-                  <div id="lineNum44" class="line heartbeat_line">beat.</div>
-               </div>
-            </div>
-
             <div id="line_info">
-
-               <div id="line_info_point1" class="line_info_point" style="display: none;"> "Empty
-                  cavern" and "cavity" refer to the ribcage of the person dying. </div>
-
-               <div id="line_info_point2" class="line_info_point" style="display: none;"> The
-                  "beeping" is referencing the sound of an EKG in a hospital. </div>
-
-               <div id="line_info_point3" class="line_info_point" style="display: none;"> The "long,
-                  wooden cage" in this line is a casket -- meaning the person dying throughout the
-                  beginning of the poem has now passed. </div>
-
-               <div id="line_info_point4" class="line_info_point" style="display: none;"> Their
-                  tombstone. </div>
-            </div>
-         </div>
-         <div class="credit"><a href="https://freesound.org/people/InspectorJ/sounds/485076/"
-               >"Heartbeat, Regular, Single, 01-01, LOOP.wav"</a> by InspectorJ (<a
-               href="www.jshaw.co.uk">www.jshaw.co.uk</a>) of <a href="http://freesound.org/"
-               >Freesound.org</a></div>
-         <div class="credit"><a
-               href="https://stackoverflow.com/questions/8655741/javascript-hide-all-other-divs"
-               >Line information toggle Javascript</a> by <a
-               href="https://stackoverflow.com/users/128165/gabriele-petrioli">Gabriele Petrioli</a>
-            on Stackoverflow.com </div>
+               
+               <div class="line_info_point-l-12 line_info_point-l-11 line_info_point-l-12" title="line_info_point" style="display: none;"><q>Empty cavern</q> and <q>cavity</q> refer to
```

Comment:
Notice here, I'm outputting multiple values so you have the information for each line that would need to be highlighted.